tinker/faculty_bios/faculty_bio_controller.py

```diff
@@ -122,21 +124,201 @@ def _iterate_child_xml(self, child, author):
             school_array = []
             for school in child.findall('.//job-titles/school'):
                 school_array.append(school.text or 'Other')
-
-            page_values = {
-                'author': child.find('author') or None,
-                'id': child.attrib['id'] or "",
-                'title': child.find('title').text or None,
-                'created-on': child.find('created-on').text or None,
-                'path': 'https://www.bethel.edu' + child.find('path').text or "",
-                'schools': school_array,
-                'last-name': child.find('.//last').text or None,
-                'deactivated': child.find('.//deactivate').text or None
+            if not csv:
```

Comment:
For simplicity, can you change this to be a `if csv:`, and of course swap the 2 sections? this keeps it simpler

Reply:
Done